recipe_scrapers/breadtopia.py

```diff
@@ -0,0 +1,42 @@
+# mypy: allow-untyped-defs
+
+from ._abstract import AbstractScraper
+
+
+class Breadtopia(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "breadtopia.com"
+
+    def author(self):
+        return self.schema.author()
+
+    def title(self):
+        return self.schema.title()
+
+    def category(self):
+        return self.schema.category()
+
+    def total_time(self):
+        return self.schema.total_time()
+
+    def yields(self):
+        return self.schema.yields()
+
+    def image(self):
+        return self.schema.image()
+
+    def ingredients(self):
+        return self.schema.ingredients()
+
+    def instructions(self):
+        return self.schema.instructions()
+
+    def ratings(self):
+        return self.schema.ratings()
+
+    def cuisine(self):
+        return self.schema.cuisine()
```

Comment:
@brett does this site support `cuisine` in the schema metadata on each/most pages?  It doesn't appear to exist for the test case.

Reply:
@jayaddison : this was my first time using the generator, and I naively assumed it would only generate methods that were supported by the site. 

I removed the erroneous methods and will update.